components/frodo/Display.cpp

```diff
@@ -533,7 +533,9 @@ static void translate_key(int key, bool key_up, uint8 *key_matrix, uint8 *rev_ma
 		case ODROID_KEY_GRAVE_ACCENT:
 			c64_key = func_flag ? MATRIX(6,3) : MATRIX(7,1); 
 			break;
-		case ODROID_KEY_BACKSLASH: c64_key = MATRIX(6,5); break;	// =
+		case ODROID_KEY_BACKSLASH:
+			c64_key = func_flag ? MATRIX(6,6) : MATRIX(6,5);
```

Comment:
This also does not do what is stated.  The backslash is mapped to '=', not 'up arrow'

Reply:
Your comment is referring to 3 commits. Please check the actual commit IDs for descriptions.

Reply:
The commit description is correct for the ODROID_KEY_BACKSLASH remapping.